macos/boot_configs/macOs_13.md

```diff
@@ -0,0 +1,151 @@
+
+```text
+    boot_screen_config {
+    boot_command     = ["<wait1s><enter>"]
+    screen_name      = "Empty"
+    matching_strings = []
+  }
+  boot_screen_config {
+    boot_command     = ["<wait1s><enter>"]
+    screen_name      = "GetStarted"
+    matching_strings = ["Get Started"]
+  }
+  boot_screen_config {
+    boot_command     = ["<wait1s><enter>"]
+    screen_name      = "GetStarted"
+    matching_strings = ["hola"]
+  }
+  boot_screen_config {
+    boot_command     = ["<wait1s><enter>"]
+    screen_name      = "GetStarted"
+    matching_strings = ["hallo"]
+  }
+  boot_screen_config {
+    boot_command     = ["<leftShiftOn><tab><leftShiftOff><spacebar>"]
+    screen_name      = "Language"
+    matching_strings = ["English", "Language", "Australia", "India"]
+  }
+  boot_screen_config {
+    boot_command     = ["<leftShiftOn><tab><leftShiftOff><spacebar>"]
+    screen_name      = "Country"
+    matching_strings = ["Select Your Country or Region"]
+  }
+  boot_screen_config {
+    boot_command     = ["<leftShiftOn><tab><leftShiftOff><spacebar>"]
+    screen_name      = "SpokenLanguages"
+    matching_strings = ["Written and Spoken Languages"]
+  }
+  boot_screen_config {
+    boot_command     = ["<leftShiftOn><tab><leftShiftOff><spacebar>"]
+    screen_name      = "Accessibility"
+    matching_strings = ["Accessibility", "Vision", "Hearing", "Motor", "Cognitive"]
+  }
+  boot_screen_config {
+    boot_command     = ["<leftShiftOn><tab><leftShiftOff><spacebar>"]
+    screen_name      = "DataAndPrivacy"
+    matching_strings = ["Data", "Privacy", "This icon appears"]
+  }
+  boot_screen_config {
+    boot_command     = ["<tab><tab><tab><spacebar>"]
+    screen_name      = "MigrationAssistant"
+    matching_strings = ["Migration Assistant", "From a Mac", "Time Machine backup"]
+  }
+  boot_screen_config {
+    boot_command     = ["<leftShiftOn><tab><tab><leftShiftOff><spacebar>"]
+    screen_name      = "SignInWithApple"
+    matching_strings = ["Sign in with your apple ID", "Sign in to use iCloud"]
+  }
+  boot_screen_config {
+    boot_command     = ["<tab><spacebar>"]
+    screen_name      = "SignInWithApplePopup"
+    matching_strings = ["Are you sure you want to skip", "signing in with an Apple ID?"]
+  }
+  boot_screen_config {
+    boot_command     = ["<leftShiftOn><tab><leftShiftOff><spacebar><wait1s><tab><spacebar>"]
+    screen_name      = "TermsAndConditions"
+    matching_strings = ["Terms and Conditions", "macOS Software Licence Agreement"]
+  }
+  boot_screen_config {
+    boot_command     = ["${local.ssh_username}<tab><tab>${local.ssh_username}<tab>${local.ssh_username}<tab><tab><tab><spacebar>"]
```

Comment:
should be ${local.ssh_username}<tab><tab>${local.ssh_password}<tab>${local.ssh_password}